### ISSUE TITLE
tabbing environment not working

### ISSUE THREAD

**Issue by Spamakin** [brucemiller/LaTeXML#2284]
When I compile the following code to a PDF, the tabbing environment works just fine. However, it is not rendering into HTML properly. Consider the following minimal tex code:
```tex
\documentclass{article}

\title{Blog Testing}
\author{Anakin Dey}

\begin{document}
\maketitle

\begin{tabbing}
  \quad\=\qquad\=\qquad\=\qquad\=\qquad\=\qquad\=\kill
  \textsc{Function}$(x)$:\+
  \\  \textbf{return} $x$
\end{tabbing}

\end{document}
```

The PDF produced using pdflatex is as follows
![image](https://github.com/brucemiller/LaTeXML/assets/60445095/b922a622-bd84-4cf6-8a82-78be5bb75bbd)

However, here is the HTML output:
![image](https://github.com/brucemiller/LaTeXML/assets/60445095/f1e1d312-39f4-495f-90c7-c54f313b2a19)

This was produced by running the following command
```sh
latexml --dest=main.xml main && latexmlpost --dest=main.html main.xml
```

**Comment by dginev** [comment]
@Spamakin just to be clear - is the issue here that the leading space before `return` is getting trimmed?

**Comment by Spamakin** [comment]
Yup that is the issue. That bolded `return` should be indented but is not.
